src/DB/projects.json

```diff
@@ -784,20 +784,20 @@
     ]
   },
   {
-    "github_username": "itsme-shivamkumar",
+    "github_username": "akkshayTandon",
     "Social_media": {
-      "gitHub": "https://github.com/itsme-shivamkumar",
-      "LinkedIn": "https://www.linkedin.com/in/shivam-kumar-9575a7227/",
-      "Instagram": "https://www.instagram.com/sherlock_shivam/",
+      "gitHub": "https://github.com/akkshayTandon",
+      "LinkedIn": "https://www.linkedin.com/in/akkshaytandon/",
+      "Instagram": "https://www.instagram.com/tandonakkshay/",
       "YouTube": "",
-      "Twitter": ""
+      "Twitter": "https://twitter.com/AkkshayTandon"
     },
     "Projects": [
       {
-        "link": "https://github.com/itsme-shivamkumar/desume",
-        "title": "Desume | Resume Builder For Developers",
-        "description": "Desume Editor is a React-based web app that allows developers to edit their resumes with ease. With Desume Editor, developers can create or modify resumes with HTML and CSS directly in the app.",
-        "tech": ["React", "Firebase", "JavaScript", "CSS", "HTML", "Framer Motion"]
+        "link": "https://github.com/akkshayTandon/C-Programs",
+        "title": "C-Programs",
+        "description": "This repository contains some useful C programs, each categorized, which can help the college students.",
+        "tech": ["C"]
```

Comment:
you deleted someone's code could you revert these changes?

Reply:
please re-review